plugins/sentinelone/plugin.spec.yaml

```diff
@@ -2598,6 +2599,77 @@ actions:
         description: SentinelOne API call response data
         type: get_events_response
         required: false
+  update_analyst_verdict:
+    title: Update Analyst Verdict
+    description: Update analyst verdict
```

Comment:
https://docs.rapid7.com/insightconnect/style-guide/#rules-2
Please follow style guide here for title and description